src/pudl/transform/ferc1.py

```diff
@@ -1962,6 +1962,75 @@ def apply_xbrl_calculation_fixes(
                     },
                 ],
             },
+            "electric_plant_depreciation_functional_ferc1": {
+                # We use this name for the calculation but it gets renamed in
+                # `process_xbrl_metadata` to total.
```

Comment:
hm yeah i think this is needed bc i never implemented a rename process for  `raw_xbrl_factoid_to_pudl_name` for the `unstack_balances_to_report_year_instant_xbrl` transform.